integrations/intercom/syncs/conversations.md

```diff
@@ -0,0 +1,75 @@
+# Conversations
+
+## General Information
+
+- **Description:** Fetches a list of conversations from Intercom
+
+- **Version:** 1.0.0
+- **Group:** Others
+- **Scopes:**: _None_
+- **Endpoint Type:** Sync
+- **Code:** [🔗](https://github.com/NangoHQ/integration-templates/tree/main/integrations/intercom/syncs/conversations.ts)
+
+
+## Endpoint Reference
+
+### Request Endpoint
+
+- **Path:** `undefined`
+- **Method:** `GET`
```

Comment:
it's broken when scripts have multiple endpoints

Reply:
Ooh, thanks. From spot checks I didn't realize that was possible.

Reply:
See e0cf4dc9 for the fix based on our discussion. Also logged https://linear.app/nango/issue/NAN-2365/support-multiple-endpoints-for-readme-generation so we can come back and updated the format for multi-endpoint readmes.